src/WebJobs.Script/Binding/FunctionBinding.cs

```diff
@@ -134,5 +121,13 @@ protected string Resolve(string name)
 
             return _config.HostConfig.NameResolver.ResolveWholeString(name);
         }
+
+        internal static void AddStorageAccountAttribute(Collection<CustomAttributeBuilder> attributes, string connection)
```

Comment:
I think that makes sense. Class-level attributes are a nice shortcut whne you have to write it yourself; but since Script is generating everything anyways, it may as well be function-level. 

---
In reply to: [56881593](https://github.com/Azure/azure-webjobs-sdk-script/pull/128#discussion_r56881593) [](ancestors = 56881593)